content/en/docs/community/release_checklist.md

```diff
@@ -265,12 +302,12 @@ git cherry-pick -x <commit_id>
 ```
 
 You will also want to update the release version number and the CHANGELOG as we
-did in steps 2 and 3 as separate commits.
+did before, as separate commits.
```

Comment:
I'm worried that various revisions mean those step numbers will get outdated. If essential, we should link to anchors.